test/runtime/fake_vqeruntime.py

```diff
@@ -18,15 +18,15 @@
 from qiskit.circuit import QuantumCircuit
 from qiskit.opflow import PauliSumOp
 from qiskit.providers import Provider
-from qiskit_nature.runtime import VQEProgramResult
+from qiskit_nature.runtime import VQERuntimeResult
 
 
 class FakeVQEJob:
     """A fake job for unit tests."""
 
     def result(self) -> Dict[str, Any]:
         """Return a VQE result."""
-        result = VQEProgramResult()
+        result = VQERuntimeResult()
```

Comment:
You should add a test which ensures that the deprecated code still works as intended as long as it is not removed, yet